### ISSUE TITLE
`padding_mode` parameter of `nn.ConvTranspose1d()`, `nn.ConvTranspose2d()` and `nn.ConvTranspose3d()` is not explained in the docs

### ISSUE THREAD

**Issue by hyperkai** [pytorch/pytorch#136072]
### 📚 The doc issue

[nn.ConvTranspose1d()](https://pytorch.org/docs/stable/generated/torch.nn.ConvTranspose1d.html), [nn.ConvTranspose2d()](https://pytorch.org/docs/stable/generated/torch.nn.ConvTranspose2d.html) and [nn.ConvTranspose3d()](https://pytorch.org/docs/stable/generated/torch.nn.ConvTranspose3d.html) has `padding_mode` parameter according to the docs as shown below:

> CLASS torch.nn.ConvTranspose1d(in_channels, out_channels, kernel_size, stride=1, padding=0, output_padding=0, groups=1, bias=True, dilation=1, padding_mode='zeros', device=None, dtype=None)

> CLASS torch.nn.ConvTranspose2d(in_channels, out_channels, kernel_size, stride=1, padding=0, output_padding=0, groups=1, bias=True, dilation=1, padding_mode='zeros', device=None, dtype=None)

> CLASS torch.nn.ConvTranspose3d(in_channels, out_channels, kernel_size, stride=1, padding=0, output_padding=0, groups=1, bias=True, dilation=1, padding_mode='zeros', device=None, dtype=None)

`padding_mode` parameter is not explained in the docs as shown below:

> Parameters
> - in_channels ([int](https://docs.python.org/3/library/functions.html#int)) – Number of channels in the input image
> - out_channels ([int](https://docs.python.org/3/library/functions.html#int)) – Number of channels produced by the convolution
> - kernel_size ([int](https://docs.python.org/3/library/functions.html#int) or [tuple](https://docs.python.org/3/library/stdtypes.html#tuple)) – Size of the convolving kernel
> - stride ([int](https://docs.python.org/3/library/functions.html#int) or [tuple](https://docs.python.org/3/library/stdtypes.html#tuple), optional) – Stride of the convolution. Default: 1
> - padding ([int](https://docs.python.org/3/library/functions.html#int) or [tuple](https://docs.python.org/3/library/stdtypes.html#tuple), optional) – dilation * (kernel_size - 1) - padding zero-padding will be added to both sides of the input. Default: 0
> - output_padding ([int](https://docs.python.org/3/library/functions.html#int) or [tuple](https://docs.python.org/3/library/stdtypes.html#tuple), optional) – Additional size added to one side of the output shape. Default: 0
> - groups ([int](https://docs.python.org/3/library/functions.html#int), optional) – Number of blocked connections from input channels to output channels. Default: 1
> - bias ([bool](https://docs.python.org/3/library/functions.html#bool), optional) – If True, adds a learnable bias to the output. Default: True
> - dilation ([int](https://docs.python.org/3/library/functions.html#int) or [tuple](https://docs.python.org/3/library/stdtypes.html#tuple), optional) – Spacing between kernel elements. Default: 1

### Suggest a potential alternative/fix

So `padding_mode` parameter should be explained in the docs as shown below:

> Parameters
> ...
> - dilation ([int](https://docs.python.org/3/library/functions.html#int) or [tuple](https://docs.python.org/3/library/stdtypes.html#tuple), optional) – Spacing between kernel elements. Default: 1
> - padding_mode ([str](https://docs.python.org/3/library/stdtypes.html#str), optional) – Only `'zeros'` can be selected. Default: 'zeros'
